test/run_sorbet.sh

```diff
@@ -101,7 +101,7 @@ fi
 if [ -z "$debug" ]; then
   command=( "bazel-bin/compiler/sorbet" )
 else
-  command=( "${LLDB:-lldb}" "--" "./bazel-bin/compiler/sorbet" )
```

Comment:
This change seems unrelated. Can we revert it?

Reply:
:shame: